scss/_variables.scss

```diff
@@ -773,7 +773,12 @@ $dt-font-weight:              $font-weight-bold !default;
 $list-inline-padding:         $spacer * .25 !default;
 
 $mark-padding:                0 .1875em !default; // Boosted mod
-$mark-bg:                     $primary !default;
+$mark-font-weight:            $font-weight-bold !default; // Boosted mod
+$mark-color:                  $white !default; // Boosted mod
+$mark-bg:                     $black !default; // Boosted mod
+
+$mark-dark-color:             $black !default; // Boosted mod
+$mark-dark-bg:                $white !default; // Boosted mod
```

Comment:
IDK what convention to choose. On one hand we have the ongoing work on Bootstrap about the Dark Mode (https://github.com/twbs/bootstrap/blob/143f94708335cb706127886db4a8d785046d1bbc/scss/_variables-dark.scss) and on the other hand we have chosen in our dark variant to do it differently.

Tried to mention it in Bootstrap: https://github.com/twbs/bootstrap/pull/35857/files#r965589842

Reply:
We talked about that with @louismaximepiton. We are going to use `$noncomponent-*-dark` for non-components and `$component-dark-*` for components with dark variants. That would be maybe more easy to handle if the naming change in the future to say in our migration guide: "dark variants components Sass vars have all been updated" and at some point another time "dark variants for non-components Sass vars have all been updated".